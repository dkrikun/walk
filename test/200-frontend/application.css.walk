#!/bin/bash

deps() {
  echo layout.css
  echo header.css
}

compile() {
  cat layout.css > application.css
  cat header.css >> application.css
}

case $1 in
  deps) deps ;;
  exec) compile ;;
esac
