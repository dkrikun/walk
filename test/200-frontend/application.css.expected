body {
  color: #000; }
#header {
  color: #fff; }

body.firefox .header:before {
  content: "Hi, Firefox users!"; }
